src/main/java/frc/robot/util/ShuffleBoard/Tabs/SubsystemTab.java

```diff
@@ -1,23 +1,30 @@
-
 package frc.robot.util.ShuffleBoard.Tabs;
 
 import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import frc.robot.commands.BangBangRotateCommand;
+import frc.robot.subsystems.DriveSub;
 import frc.robot.util.ShuffleBoard.ShuffleBoardTabs;
 
 
 public class SubsystemTab extends ShuffleBoardTabs {
     // TODO 2.3.11: Create variable for subsystem
+    
+    public DriveSub driveSub;
 
-    public SubsystemTab(){
-        
+    public SubsystemTab(DriveSub driveSub){
+        this.driveSub = driveSub;
     }
-
+    
     public void createEntries(){
         tab = Shuffleboard.getTab("Subsystem");
 
         // TODO 2.4.7: Add Mechanism2d
+        tab.add("Motor Visualization", driveSub.getMechanism());
+
+
 
         // TODO 3.3.13: Add command buttons
+        tab.add("Spin Motor", new BangBangRotateCommand(driveSub, 5)); // spin motor 5 rotations
```

Comment:
Add your BangBangSubsystemCommand, either here or in AutoTab.

Reply:
Why does it set it to 1 degree?

Reply:
Maybe change it to 90 or 180 degrees, or another angle that is far enough to notice the change.

Reply:
I was trying to change everything to *rotations* (360 degrees), I changed the *Bang Bang Command*, which hopefully runs in rotations.